drivers/shared/executor/executor.go

```diff
@@ -609,19 +623,23 @@ func makeExecutable(binPath string) error {
 	return nil
 }
 
-func (e *UniversalExecutor) cleanupHandle() []byte {
+func (e *UniversalExecutor) cleanupHandle() ([]byte, error) {
+	startTime, err := processStartTime(int32(e.childCmd.Process.Pid))
+	if err != nil {
+		return nil, err
```

Comment:
given that process start time is a heuristic for identifying pid recycling - I'd rather consider relying on other heauristics rather than give up completely on cleaning up.

Reply:
I decided to punt on this for another PR.